charts/service-deployment/README.md

```diff
@@ -43,6 +43,7 @@ helm delete service-deployment
 
```

Comment:
It looks like there are quite a few catchup changes here from previous releases ?

Reply:
Indeed a lot were missing so have added them in.

Reply:
I used `helm-docs -d` to generate this automatically which is why some of the order has changed - it seems we have been adding new options just to the bottom so this brings the sort order back into play.